documents/translation-workflow.md

```diff
@@ -46,34 +46,48 @@ The purpose of this document is to explain the general workflow and commands req
 
     ```sh
     npm run translate:extract
+    ```
+    
+   -	Upload & download. This step uploads the text strings to Crowdin Platform (To be translated by translators) and downloads the latest `ach tags`.
+   
+   ```sh
+    npm run translate:upload
+    npm run translate:download
     ```
 
--   Commit your changes
+	 -	Commit your changes
 
 5. Make pull request
 
    -   Push your changes to your origin, add `-u` flag for the first time push
-
-       ```sh
-       git push -u origin [_your_branch_name]
-       ```
+   
+   ```sh
+   git push -u origin [_your_branch_name]
+   ```
 
    -   Click on the autogenerated link from terminal to open the PR
 
    -   Make sure to change the PR base to `crowdin` branch
 
-6. Uploading source text to crowdin platform (To be translated by translators)
 
-    ```sh
-    npm run translate:upload
-    ```
+6. Download/get latest crowdin translations **(Once strings are translated on Crowdin Platform)**.
 
-7. Download/get latest crowdin translations (Once strings are translated on Crowdin Platform). 
     ```sh
     npm run translate:download
     ```
-    This triggers a build and redeploys an updated version of the crowdin test link with the new translated texts.
+    
+7. Make pull request, get reviewed & merged (This triggers a build and redeploys an updated version of the crowdin test link with the new translated texts).
 
 8. The task now goes through a translation regression check to test for all other languages.
 
-9. Once ready, it can be merged into master awaiting to be rebased.
\ No newline at end of file
+9. Once ready, the crowdin branch can be merged into master awaiting to be rebased.
+
+### Merging from Crowdin to Master & Rebasing:
+
+1. Create a PR from Crowdin to Master.
+2. Once approved and merged, the staging link will need to be tested for translations.
+3. Rebase Crowdin branch onto master & delete any irrelevant commits.
+⚠️ Note: Be very careful here and do take note to compare with the commits on master.
+   ```sh
+    git rebase --interactive
+   ```
```

Comment:
maybe we should add some informations about :

Step 3. pick --> d ( if it's not your commit )  - press i to insert , esc and :wq to exit
Step 4. git add . 
Step 5. git rebase --continue
Step 6. git push -f later after finish rebasing

if rebase failed --> rebase --abort

**make a backup of the folder also good / jump to their previous commit 

** failing to merge can somehow close the PR automatically

Reply:
Sounds like a good idea 👍  Could you help to add it to the wiki.js first? This PR was initially made to update the info as the previous one was actually incorrect. We can finalise the update to the wiki first before I copy it over to the docs here then.

Reply:
Okay, I'll add it to wiki.js , I missed your comment previously